packages/webapp/src/containers/Task/index.jsx

```diff
@@ -99,13 +98,9 @@ export default function TaskPage({ history }) {
   }, [activeUsers.length, taskTypes.length]);
 
   useEffect(() => {
-    dispatch(getTasks());
-    dispatch(getTaskTypes());
-    dispatch(getProducts());
-    dispatch(getHarvestUseTypes());
     dispatch(getAllUserFarmsByFarmId());
+    dispatch(getManagementPlansAndTasks());
```

Comment:
I've tested it with this PR, and this is the change that fixes my current ticket -- with the previous code we were fetching the tasks first and the management plans at the end, here we're fetching all of the dependencies first and then the tasks